{
  "libname": "arcgis",
  "libvers": "2.4.1",
  "infile": "data/pip/arcgis.txt",
  "dependencies": {
    "annotated-types": "0.7.0",
    "cachetools": "5.5.2",
    "certifi": "2025.4.26",
    "cffi": "1.17.1",
    "charset-normalizer": "3.4.2",
    "click": "8.1.8",
    "cloudpickle": "3.1.1",
    "contourpy": "1.3.2",
    "cryptography": "44.0.3",
    "cycler": "0.12.1",
    "dask[array,dataframe]": "2024.12.1",
    "dask-expr": "1.1.21",
    "fonttools": "4.57.0",
    "fsspec": "2025.3.2",
    "geomet": "1.1.0",
    "idna": "3.10",
    "jaraco-classes": "3.4.0",
    "jaraco-context": "6.0.1",
    "jaraco-functools": "4.1.0",
    "keyring": "25.6.0",
    "kiwisolver": "1.4.8",
    "locket": "1.0.0",
    "lxml": "5.4.0",
    "matplotlib": "3.10.1",
    "matplotlib-inline": "0.1.7",
    "more-itertools": "10.7.0",
    "networkx": "3.4.2",
    "numpy": "1.26.4",
    "oauthlib": "3.2.2",
    "packaging": "25.0",
    "pandas": "2.2.3",
    "partd": "1.4.2",
    "pillow": "11.2.1",
    "puremagic": "1.29",
    "pyarrow": "16.1.0",
    "pycparser": "2.22",
    "pydantic": "2.11.4",
    "pydantic-core": "2.33.2",
    "pylerc": "4.0",
    "pyparsing": "3.2.3",
    "pyspnego": "0.11.2",
    "python-dateutil": "2.9.0.post0",
    "pytz": "2025.2",
    "pyyaml": "6.0.2",
    "requests": "2.32.3",
    "requests-oauthlib": "2.0.0",
    "requests-toolbelt": "1.0.0",
    "six": "1.17.0",
    "toolz": "1.0.0",
    "traitlets": "5.14.3",
    "truststore": "0.10.1",
    "typing-extensions": "4.13.2",
    "typing-inspection": "0.4.0",
    "tzdata": "2025.2",
    "ujson": "5.10.0",
    "urllib3": "2.4.0",
    "websocket-client": "1.8.0"
  }
}